pmlc/dialect/pxa/tests/autotile.mlir

```diff
@@ -12,5 +11,16 @@ func @dot(%arg0: memref<100x100xf32>, %arg1: memref<100x100xf32>) -> memref<100x
   }
   return %out : memref<100x100xf32>
 }
-// CHECK: affine.parallel
-// CHECK: affine.parallel
+
+// CHECK-LABEL: func @dot
+// CHECK-SAME: (%arg0: memref<100x100xf32>, %arg1: memref<100x100xf32>) -> memref<100x100xf32>
```

Comment:
%arg0 should also be captured, or just use `%{{.*}}` if you don't need a capture